src/main/kotlin/fi/metatavu/timebank/api/severa/models/SeveraUser.kt

```diff
@@ -0,0 +1,24 @@
+package fi.metatavu.timebank.api.severa.models
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties
+import com.fasterxml.jackson.annotation.JsonProperty
+import io.quarkus.runtime.annotations.RegisterForReflection
+
+/**
+ * Data class for User data coming from Severa
+ */
+@RegisterForReflection
+@JsonIgnoreProperties(ignoreUnknown = true)
+class SeveraUser {
+    var guid: String = ""
+    @JsonProperty("firstName")
```

Comment:
You don't need `@JsonProperty` if the JSON field has the exact same name as the POJO does.

Reply:
This doesn't seem to be changed?